server/API DOCS/GET.md

```diff
@@ -17,5 +17,24 @@
     * **Code:** `401` <br />
       **Message:** `authorization required` <br />
 
+* **Required Filed**
+    * **Auth/ Admin Auth: true/true**
+
+
+**2. Get answers**
+* **URL**
+    `/api/getAnswers/:id`
+
+* **METHOD**
+    `GET`
+
+* **Success Response**<br />
+    * **Code:** `200` <br />
+      **data:** `{ answers: [ displaying all answers to a particular question] }`
+
+* **Error Response**
+    * **Code:** `400` <br />
+      **Message:** `something went wrong` <br />
+
```

Comment:
Here not need to be admin auth so you can write like this.

Auth/ Admin Auth: true/false (First for user Auth/second for admin auth)

Reply:
I have made a change, is that fine?

Reply:
Yup!